plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/annotationProcessing.kt

```diff
@@ -142,6 +144,17 @@ private fun showProcessorTimings(wrappedProcessors: List<ProcessorWrapper>, logg
     }
 }
 
+private fun dumpProcessorTiming(wrappedProcessors: List<ProcessorWrapper>, apReportFile: File?, logger: (String) -> Unit) {
+    logger("Dumping Kapt Annotation Processing performance report to ${apReportFile?.absolutePath}")
+
+    apReportFile?.writeBytes(buildString {
```

Comment:
File.writeText maybe?

Reply:
Updated.
Note that I did it this way because that's how -Xdump-perf works (see [code](https://github.com/JetBrains/kotlin/blob/9b6f95faa257cf750726da5eff3e57e6b8ec6f85/compiler/cli/src/org/jetbrains/kotlin/cli/common/CommonCompilerPerformanceManager.kt#L153))